README.md

```diff
@@ -557,6 +557,41 @@ application: litestar.Litestar = (
 >
 > In this case, Prometheus will receive `{"first_value": 1, "second_value": 2}` inside `prometheus_additional_params`. This is also true for `list`, `tuple`, and `set`.
 
+### Using microbootstrap without a framework
+
+You can use microbootstrap to set up Sentry, OpenTelemetry and Logging with `InstrumentsSetupper`:
+
+```python
+from microbootstrap.instruments_setupper import InstrumentsSetupper
+from microbootstrap import InstrumentsSetupperSettings
+
+
+class YourSettings(InstrumentsSetupperSettings):
+    ...
+
+
+setupper = InstrumentsSetupper(YourSettings())
```

Comment:
I think configurator sounds more appealing then setupper = ... Setupper. I recommend naming variables so that they are different from what they correspond to.

Reply:
I don't want to introduce another synonym, so I renamed into `current_setupper`